shared_model/bindings/simple_builder.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_SIMPLE_BUILDER_HPP
+#define IROHA_SIMPLE_BUILDER_HPP
+
+#include "builders/protobuf/proto_transaction_builder.hpp"
+
+namespace shared_model {
+  namespace proto {
+    class SimpleBuilder {
```

Comment:
I request to add documentation and write a note about this code just for bindings.